Automattic/wp-calypso
# Media: Dispatch validation error redux actions from flux actions

#### Changes proposed in this Pull Request

This PR:

* Introduces new media validation actions where necessary wrt validation errors
* Changes validation store so it dispatches validation redux actions when the corresponding flux actions are dispatched.

#### Testing instructions

* Checkout this branch.
* Verify all tests pass.
* Try smoke testing media library and verify it still works well.

Note: This PR is part of an iteration to remove the media validation store, see #39372 for the full effort that removes it completely.
